global.json

```diff
@@ -2,6 +2,6 @@
   "sdk": {
     "version": "8.0.100",
     "allowPrerelease": false,
-    "rollForward": "disable"
+    "rollForward": "latestFeature"
```

Comment:
Unrelated to the issue at hand, but I needed to change this to be able to build under the following configuration:
![image](https://github.com/nunit/nunit/assets/439844/28907270-dda9-46ff-9667-01091417762f)

Reply:
You can set `"rollForward": "latestPatch"` to allow 8.0.1xx.
Now it also allows 8.0.2xx and MS is has in the past broken things when adding new "features". (6.0.300)
But I haven't found anything breaking with 8.0.200

Reply:
Thanks! I hadn't realized there was a precedent there.